twitter/View Controllers/TimelineViewController.m

```diff
@@ -47,7 +47,18 @@ - (void)fetchTweets {
      (NSArray<Tweet *> *tweets, NSError *error) {
         if (tweets) {
             for (Tweet *tweet in tweets) {
-                [self.tweetModels addObject:[[TweetCellModel alloc] initWithTweet:tweet]];
+                [self.tweetModels addObject:[[TweetCellDecorator alloc] initWithTweet:tweet]];
+            }
+            [self.homeTweetTableView reloadData];
+        }
+    }];
+}
+
+- (void)fetchMoreTweets:(NSString *)tweetId {
+    [[APIManager shared] getMoreHomeTimelineTweets:tweetId completion:^ (NSArray<Tweet *> *tweets, NSError *error) {
+        if (tweets) {
+            for (int i = 1; i < tweets.count; i++) {
```

Comment:
For tweet in tweet? I personally find this loop syntax really hard to reason about and a big source of bugs.